scripts/gee

```diff
@@ -4127,6 +4125,21 @@ Tested:
 
 EndOfPrTemplate
 
+    local OWNERS=()
+    mapfile -t OWNERS < <(_print_codeowners)
+    if [[ "${#OWNERS[@]}" -eq 0 ]]; then
+      printf "# This PR can be submitted with anyone's approval.\n"
+    elif [[ "${#OWNERS[@]}" -eq 1 ]]; then
+      printf "# This PR requires approval from at least one of these code owners:\n"
+      printf "#  * %s\n" "${OWNERS[@]}"
+    else
+      printf "# This PR requires approval from at least one code owner on each line:\n"
+      printf "#  * %s\n" "${OWNERS[@]}"
+    fi
+    local UNIQUE_OWNERS="$(echo "${OWNERS[*]}" | xargs -n1 | sort -u | xargs | sed -e 's/@//g' )"
+    printf "# Reduce this set of assignees:\n"
+    printf "ASSIGNEES: ${UNIQUE_OWNERS}\n\n"
```

Comment:
I have a strong preference for not defaulting this.

We have to use `assignees` and not `reviewers` due to how Github defaults the latter, which creates too much noise.

Defaulting this field to a non-minimal set has the potential to recreate noise. Once `assignees` is sufficiently polluted, there are no more Github fields to leverage.

Put a different way - if I'm assigned every single bzl review "by default", I will stop using github_badger.

Reply:
OK, done.